kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaCommandDecoder.java

```diff
@@ -650,8 +655,17 @@ public String toString() {
                 this.header, this.request, this.remoteAddress);
         }
 
+        public void bufferReleased() {
+            if (!released.compareAndSet(false, true)) {
```

Comment:
I'm not sure about this method, there is no release operation but changing the released state.

Reply:
Good point, I find we don't need to use this method. They are using this method on the proxy module.